pkg/test/deps/istio.go

```diff
@@ -0,0 +1,50 @@
+//  Copyright 2019 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package deps
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"istio.io/istio/pkg/test/env"
+)
+
+var (
+	Istio []IstioDep
+)
+
+func init() {
+	content, err := ioutil.ReadFile(filepath.Join(env.IstioSrc, "istio.deps"))
```

Comment:
Apparently this is now a pointless/misleading file since we moved to go modules. I think the source of truth is the `go.mod` file now. Is that right @fpesce @duderino ?

Reply:
I don't see any reference to istio proxy in `go.mod` ... I suspect we still need the `istio.deps` file.

@fpesce @duderino can you confirm?

Reply:
go.mod is our source of truth for checking the api dependency (and comparing, during release process, to the one used in istio/proxy too).

We still need the istio.deps as a source of truth for the proxy dependency.

Reply:
go.mod only deals with Go code. So it definitely will not deal the proxy dependency.